experimental/sql/src/impl_to_iterators.py

```diff
@@ -30,6 +32,15 @@ def convert_datatype(self, type_: RelImpl.DataType) -> Attribute:
     if isinstance(type_, RelImpl.Int64):
       # TODO: this is obviously broken, but the backend currently only support i32
       return IntegerType.from_width(32)
+    if isinstance(type_, RelImpl.Decimal):
+      return IntegerType.from_width(32)
+    if isinstance(type_, RelImpl.Timestamp):
+      return IntegerType.from_width(32)
+    if isinstance(type_, RelImpl.String):
+      return LLVMStructType([
+          StringAttr.from_str(""),
+          ArrayAttr.from_list([IntegerType.from_width(8)] * 8)
```

Comment:
I think that this deserves a comment (being a shortcut, limitations, etc.). Maybe also a runtime error if longer strings are encountered...

Reply:
Currently, there is an error if any string is encountered in the LiteralRewriter. It only matches on Ints, Decimals and Timestamp, but not on Strings. 

I will add the comment :)